src/views/SearchSelfCareView.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from "react";
+import { Helmet } from "react-helmet";
+import SelfCareSearchBar from "../components/searchSelfCareView/SelfCareSearchBar";
+
+const SearchSelfCareView = () => {
+  const [searchResults, setSearchResults] = useState({});
+
+  return (
+    <>
+      <Helmet>
+        <title>Search Self Care</title>
```

Comment:
This is not really a user-friendly formulation, I see that you also use that in the search field as a placeholder. Did we have that on the wireframes?

Reply:
Search for Self Care services maby?

Reply:
placeholder: Search for a service...
ok?

Reply:
Yeah, I think it is. 👍🏼